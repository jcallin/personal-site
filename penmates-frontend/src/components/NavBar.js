import React from "react";
import { Link } from "react-router-dom";

function NavBar() {
  return (
    <nav className="navbar navbar-expand-lg navbar-light">
      <Link className="navbar-brand" to="/ ">
        Julian Callin
      </Link>
      <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
        <div className="navbar-nav">
          <a className="nav-item nav-link" href="#">
            Resume
          </a>
          <a className="nav-item nav-link" href="#">
            Projects
          </a>
          <a className="nav-item nav-link" href="#">
            Pictures
          </a>
        </div>
      </div>
    </nav>
  );
}

export default NavBar;
