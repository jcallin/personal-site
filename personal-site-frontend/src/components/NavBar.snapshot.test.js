import React from 'react'
import { shallow } from 'enzyme'
import NavBar from './NavBar'
describe('NavBar component', () => {
  it('should have a matching snapshot', () => {
    const wrapper = shallow(<NavBar />)
    expect(wrapper).toMatchSnapshot()
  })
})
