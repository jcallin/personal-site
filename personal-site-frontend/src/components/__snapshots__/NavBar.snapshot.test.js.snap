// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component should have a matching snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Link
        className="navbar-brand"
        replace={false}
        to="/"
      >
        Penmates
      </Link>,
      "className": "navbar navbar-dark bg-primary fixed-top",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "Penmates",
        "className": "navbar-brand",
        "replace": false,
        "to": "/",
      },
      "ref": null,
      "rendered": "Penmates",
      "type": [Function],
    },
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Link
          className="navbar-brand"
          replace={false}
          to="/"
        >
          Penmates
        </Link>,
        "className": "navbar navbar-dark bg-primary fixed-top",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Penmates",
          "className": "navbar-brand",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": "Penmates",
        "type": [Function],
      },
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
